### PR TITLE
Forget about me feature + follow fix

### PR DESCRIPTION
**Pull request by tomat-suppe**
New functionalities:
* Forget me button from Usertimeline: deleted logged in user and all of their cheeps
* - Also deleted logged in user from other user's followlist, if they followed deleted user.
* Followbutton no longer appears on logged in user, and others can be unfollowed from private timeline.
Minor fix
* usings alphabetic order
* deleted irrelevant comments 
* deleted outcommented code (here and there, not every class who has outcommented code)

Co-authored-by: Henriette <hoes@itu.dk>
Co-authored-by: Sophie <slak@itu.dk>

### CODE REVIEW

**Review by Thyme-Cat — APPROVED** [review]
Everything looks very good! I have left one minor comment, but otherwise it is good to go! XD

**Review by JaSNOHo — APPROVED** [review]
lgtm

File: src/ChirpInfrastructure/AuthorRepository.cs [hunk 1]
```diff
@@ -39,53 +41,78 @@ public void LoginAuthor() { }
...
 
 	public async Task<Author> GetAuthorFromUsername(string? Username)
 	{
-		if (Username == null){
+		if (Username == null)
+		{
 			throw new ArgumentNullException(Username);
 		}
-		return await _context.Authors.Include(a=>a.Follows).Where(Author => Author.UserName == Username).FirstAsync();	
+		return await _context.Authors.Include(a => a.Follows).Where(Author => Author.UserName == Username).FirstAsync();
```
Comment (HenrijetteOest) [inline src/ChirpInfrastructure/AuthorRepository.cs:64]: forslag pga ovenover: (A => A.Follows).Where(Author => Author.UserName == Username).FirstAsync(); eller no?

**Review by llsa-itu — APPROVED** [review]

File: src/ChirpInfrastructure/AuthorRepository.cs [hunk 2]
```diff
@@ -39,53 +41,78 @@ public void LoginAuthor() { }
 
 	//this method is used when an Author unfollows another Author
 
-	public void DeleteAuthorFromDatabase() { }
+	public async Task DeleteAuthorFromDatabase(string Username)
```
[the remark this answers is not in the document]
Comment (tomat-suppe) [reply src/ChirpInfrastructure/AuthorRepository.cs]: updated and resolved <3

File: src/ChirpInfrastructure/AuthorRepository.cs (hunk 1 above)
Comment (tomat-suppe) [reply src/ChirpInfrastructure/AuthorRepository.cs]: fixed

**Pull request merged**